server/server.py

```diff
@@ -3,32 +3,74 @@
 import tornado.websocket
 import tornado.template
 
+# WS_HANDLERS maintains a list of currently-opened ws connections
+WS_HANDLERS = []
+
+# maintain message history for new connections
+MESSAGE_HISTORY = []
+
 class MainHandler(tornado.web.RequestHandler):
-  def get(self):
-    loader = tornado.template.Loader(".")
-    self.write(loader.load("../client/index.html").generate())
+    def get(self):
+        loader = tornado.template.Loader(".")
+        self.write(loader.load("../client/index.html").generate())
```

Comment:
Probably something I can get answered with more Googling, but should we be using templating?  Is this the correct way to serve up our client-side html?

I'm wondering if we should just group all of this in a single directory.  I'm having a _lot_ of problems serving static CSS files right now.  I've tried 3-4 strategies to no avail yet.

(this is non-blocking for your PR as it is not relevant, just something I'd like to discuss)